crates/control/src/motion/walking_engine.rs

```diff
@@ -378,8 +378,8 @@ impl WalkingEngine {
             right_leg: LegJoints::fill(leg_stiffness),
         };
 
-        Ok(MainOutputs {
-            walk_motor_commands: MotorCommands {
+        let walk_joints_commands = {
+            MotorCommands {
```

Comment:
Please revert changes in this file. This has nothing to do with this PR

Reply:
Done